api/src/pcapi/core/offers/repository.py

```diff
@@ -823,6 +831,18 @@ def is_non_free_offer_subquery(offer_id: int) -> sa.sql.selectable.Exists:
     )
 
 
+def get_pending_bookings_subquery(offer_id: int) -> sa.sql.selectable.Exists:
```

Comment:
Est-ce qu'on ne la rendrait pas privée ?
```suggestion
def _get_pending_bookings_subquery(offer_id: int) -> sa.sql.selectable.Exists:
```

Reply:
J'y ai pas pensé vu que ce n'est pas dans une classe et qu'il y a des fonction similaire qui ne sont pas privée.